scripts/restore.sh

```diff
@@ -103,6 +109,9 @@ if [ -f "$BACKUP_FILE" ]; then
             # Decompress the backup file in tmp directory
             tar -zxvf "$BACKUP_FILE" -C "$TMP_PATH"
 
+            # Make sure Saves with a different ID are removed before restoring the save
+            rm -rf "$RESTORE_PATH/Saved/"
+
```

Comment:
Instead of deleting the directory should we instead change `\cp -rf "$RESTORE_PATH/Saved" "$TMP_SAVE_PATH/Saved"` to `mv "$RESTORE_PATH/Saved" "$TMP_SAVE_PATH/Saved"`?

Reply:
I tried your suggestion, but I get an Error if we only use mv 
```
mv: cannot move '/palworld/backups/tmp.D0rNX7pZLE/Saved/' to '/palworld/Pal/Saved': Directory not empty
An error occurred during restore.
```

Reply:
I'm guessing you did line 118?

I meant 96.

Although let's keep it simple and just use the rm you added. I'd like to the the issue as have fixed before we try to change a cp to mv because that would require more testing.

Reply:
Oww.., okay. Thought you meant the line 118. You're right, that would work. My bad.